src/interfaces/wallet.h

```diff
@@ -95,7 +95,8 @@ class Wallet
     //! Look up address in wallet, return whether exists.
     virtual bool getAddress(const CTxDestination& dest,
         std::string* name = nullptr,
-        isminetype* is_mine = nullptr) = 0;
+        isminetype* is_mine = nullptr,
+        std::string* purpose = nullptr) = 0;
```

Comment:
This list of optional arguments slightly scares me. How much diff would it be to remove the defaults and change all the places this is called to pass in nullptr?

Reply:
There are ~9 replacements this'd entail, so not too bad. Willfix.